### DISCUSSION TITLE
How to to turn economy on

### DISCUSSION THREAD

**Discussion by UlizesR** [TownyAdvanced/Towny#5624] [Q&A]
Hi, I have the EssentialsX plugin install in my server but I need to turn economy on apparently and I am not really sure how to, can someone explain to me how to do so. Thank you very much/

**Comment by ggvbo** [comment]
Hello! 
- Do you have Vault or Reserve installed? They are required as they connect economy plugins to Towny.
- EssEco is not recommended due to UUID related issues.

**Reply by UlizesR** [reply]
Thanks

**Reply by ggvbo** [reply]
No problem, remember to mark the comment that helped you the most as the answer.

**Comment by LlmDl** [comment]
Towny's economy is turned on and off by the config's using_economy setting.

Vault or Reserve is required for any economy plugin to work. EssentialsXEconomy is not recommended.

**Reply by UlizesR** [reply]
I only have EssentialX but not Vault or Reserve. Thanks!

**Comment by LlmDl** [comment]
File a bug report ticket so we can see the server files and startup
